### DISCUSSION TITLE
PSRule.Rules.Azure/concepts/policy-as-rules/

### DISCUSSION THREAD

**Discussion by giscus** [Azure/PSRule.Rules.Azure#2865] [Documentation]
# PSRule.Rules.Azure/concepts/policy-as-rules/

This topic covers using policy as rules which allows you to use Azure Policy as rules to test Infrastructure as Code.

https://azure.github.io/PSRule.Rules.Azure/concepts/policy-as-rules/

**Comment by ReneRebsdorf** [comment]
Looks exciting! Interesting shift left

**Comment by AwangkuAkram** [comment]
Hi! Do you have any ps-rule.yaml example for this? Id like to test my bicep file against my Azure Policies without Azure W.A.F but Im unable to get it to work :')

**Reply by BernieWhite** [reply]
Good call @AwangkuAkram, we should add this to the docs.

`ps-rule.yaml` will require a binding configuration similar to other custom rule scenarios. See: https://azure.github.io/PSRule.Rules.Azure/customization/enforce-custom-tags/#binding-type

In terms of only running policy rules you will still need to use the `PSRule.Rules.Azure` module however you can use the baseline mentioned [here](https://azure.github.io/PSRule.Rules.Azure/concepts/policy-as-rules/#generated-baseline) to limit the rules.
